typescript/infra/config/environments/dev/core.ts

```diff
@@ -1,24 +1,20 @@
-import { CoreConfig } from '../../../src/core';
-
-const validatorManagers = {
-  alfajores: {
-    validators: [
-      '0x4455f6B4c341d48ef8CDBe1b9bE8bb3a63c97a46',
-      '0xD3f317f27D71b2A5fF9A1Ee78a1230390f77e714',
-      '0x2C503aF4fe1BCb774E842BC0ACaAC5120dDFA560',
-    ],
-    threshold: 2,
+export const core = {
```

Comment:
This isn't typed as `CoreConfig` because TS can infer the type from somewhere else?